modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/SleepFragment.kt

```diff
@@ -114,6 +112,51 @@ class SleepFragment : BaseDialogFragment() {
             }
         }
 
+        binding.customStepperComposeView.setContentWithViewCompositionStrategy {
+            AppTheme(theme.activeTheme) {
+                NumberStepper(
+                    onMinusClick = {
+                        minusButtonClicked()
+                    },
+                    onPlusClick = {
+                        plusButtonClicked()
+                    },
+                    minusContentDescription = LR.string.player_sleep_custom_minus,
+                    plusContentDescription = LR.string.player_sleep_custom_plus,
```

Comment:
I think all of these content descriptions could be more descriptive. Something like `Increase custom duration`, `Increase chapter count`, etc. Otherwise it's doesn't help to know what the buttons do when seeing/hearing `minus` and `plus`.